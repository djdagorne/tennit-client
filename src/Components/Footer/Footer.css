footer {
    background-color: #aaa;
    position:fixed;
    bottom:0;
    left:0;
    right:0;
    width:100%;
}
  
footer a{
    margin:1vw;
}