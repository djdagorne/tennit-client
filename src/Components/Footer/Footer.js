import React from 'react';
import './Footer.css';

function Footer() {
  return (
    <footer className="footer">
        <a href="mailto:dj.dagorne@gmail.com">email</a>
        <a href="httsp://www.github.com/djdagorne">github</a>
        <a href="https://www.linkedin.com/in/djdagorne/">linkedin</a>
    </footer>
  );
}

export default Footer;
