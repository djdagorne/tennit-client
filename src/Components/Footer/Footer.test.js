import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './Footer';
import '../../setupTest';

it('renders without crashing', () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
})
