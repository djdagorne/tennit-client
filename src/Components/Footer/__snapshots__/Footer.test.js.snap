// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Fragment>
  <i
    alt="email me here"
    aria-hidden="true"
    className="text-shadow fas fa-envelope-square 2x"
    href="mailto:dj.dagorne@gmail.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    <span
      className="sr-only"
      hover="email me here"
    >
      email
    </span>
  </i>
  <i
    alt="check my github here"
    aria-hidden="true"
    className="text-shadow fab fa-github-square 2x"
    href="https://github.com/djdagorne"
    rel="noopener noreferrer"
    target="_blank"
  >
    <span
      className="sr-only"
      hover="check my github here"
    >
      github
    </span>
  </i>
  <i
    alt="check my linkedin here"
    aria-hidden="true"
    className="text-shadow fab fa-linkedin 2x"
    href="https://www.linkedin.com/in/dexter-dagorne-81461a18a/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <span
      className="sr-only"
      hover="check my linkedin here"
    >
      linkedin
    </span>
  </i>
</Fragment>
`;
